From 3fb8828b3ca0369ed9daf49dd4892bcb4e5b8c6b Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Tue, 30 Oct 2018 09:44:20 +0100
Subject: [PATCH] Merge pull request #236 in SW_UGW/linux from
 bugfix/DRVLIB_SW-961-Disable_CQEM_ENQ_backpressure to master

* commit '9356eabfcce39135e712dccfb7ab0f88d3c5318f':
  DRVLIB_SW-961:Disable CQEM DMA ENQ Backpressure
---
 drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c b/drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c
index 567fc765677d..be86abe92fcf 100644
--- a/drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c
+++ b/drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c
@@ -3083,17 +3083,14 @@ static void init_cqm_enq_dma_port(int idx, s32 type)
 			   NET_IP_ALIGN +
 			   NET_SKB_PAD +
 			   CQM_POOL_METADATA);
-	/*enable backpressure*/
 	cbm_w32(enq + EQ_DMA_PORT(idx, cfg),
 		CFG_DMA_IGP_4_EQREQ_MASK |
-		CFG_DMA_IGP_4_EQPCEN_MASK |
-		CFG_DMA_IGP_4_BP_EN_MASK);
+		CFG_DMA_IGP_4_EQPCEN_MASK);
 
 	dev_dbg(cqm_ctrl->dev, "0x%x\n", (int)enq + EQ_DMA_PORT(idx, cfg));
 
 	dev_dbg(cqm_ctrl->dev, "0x%x\n", CFG_DMA_IGP_4_EQREQ_MASK |
-		  CFG_DMA_IGP_4_EQPCEN_MASK |
-		  CFG_DMA_IGP_4_BP_EN_MASK);
+		  CFG_DMA_IGP_4_EQPCEN_MASK);
 }
 
 static void setup_deq_dma_desc(u32 pid, u32 desc_num)
